src/query/storage/converter.go

```diff
@@ -226,6 +226,8 @@ func seriesAttributesForGraphiteSource(series prompb.TimeSeries) (ts.SeriesAttri
 		promMetricType = ts.PromMetricTypeCounter
 	case prompb.M3Type_M3_GAUGE:
 		promMetricType = ts.PromMetricTypeGauge
+	case prompb.M3Type_M3_TIMER:
+		promMetricType = ts.PromMetricTypeUnknown
```

Comment:
nit: Just thinking, do we care what the prom metric type is if we are using source type graphite here?  Just thinking this could be simplified to just have `PromType` as unknown if it's `SourceTypeGraphite` and we already set the `M3Type`?

Not super important though, doubt it will effect much since we'll ignore the value of prom metric type if source type is graphite when we read the annotation.

Reply:
Hmm, interesting. I see that if we would always add both `__m3_type__` and `__m3_prom_type__` tags here:
https://github.com/m3db/m3/blob/master/src/cmd/services/m3coordinator/downsample/metrics_appender.go#L193-L218 
Even in case of `PromMetricTypeUnknown`.

Also, because of https://github.com/m3db/m3/blob/master/src/query/storage/converter.go#L224-L228, we would additionally set `__m3_prom_type__=counter` for Graphite counter (and `__m3_prom_type__=gauge` for Graphite gauge).

No idea what is the implication of this and whether there is any issue. @shreyassrivatsan could you please take a look?

Reply:
It is hard to say what the effect is - but having both the tags there maybe problematic. These things are primarily used to match rollup / mapping rules. Having both may mean graphite metrics match non-graphite rules and vice versa..